gerousiabot/bot_handlers.py

```diff
@@ -0,0 +1,80 @@
+from telegram import Update, InlineKeyboardButton, InlineKeyboardMarkup
+from telegram.ext import CommandHandler, MessageHandler, Filters, CallbackQueryHandler, CallbackContext
+
+
+def get_bot_handlers() -> list:
+    """
+    Gets the list of the handlers to be used by the bot
+    :return: the list of the handlers
+    """
+    start_handler = CommandHandler('start', start)
+    user_query_handler = CommandHandler('chkusers', check_users)
+    button_handler = CallbackQueryHandler(keyboard_button_pressed)
+    unknown_handler = MessageHandler(Filters.command, unknown_command)
+
+    # unknown command handler must be the last added handler or we run into logical errors
+    return [start_handler, button_handler, user_query_handler, unknown_handler]
+
+
+def start_keyboard() -> list:
+    """Initialize the keyboard"""
+    keyboard = [InlineKeyboardButton("Check Users", callback_data='user_check'),
+                InlineKeyboardButton("Blank button", callback_data='blank_button')]
+    return keyboard
+
+
+def keyboard_button_pressed(update: Update, context: CallbackContext) -> None:
+    """Parses the CallbackQuery for keyboard buttons"""
+    query = update.callback_query
+    # CallbackQueries need to be answered, even if no notification to the user is needed
+    # Some clients may have trouble otherwise. See https://core.telegram.org/bots/api#callbackquery
+    query.answer()
+
+    query.edit_message_text(text=f"Selected option: {query.data}")  # for debugging, remove once final
+    if query.data == "user_check":
```

Comment:
[Suggestion] After you convert the `query.data` to `enums` let's replace the `if` check with `switch` cases.
I'm assuming python has good support for switch x enums